shell/platform/darwin/macos/framework/Headers/FlutterPluginRegistrarMacOS.h

```diff
@@ -36,11 +36,16 @@ FLUTTER_DARWIN_EXPORT
 @property(nonnull, readonly) id<FlutterTextureRegistry> textures;
 
 /**
- * The default view displaying Flutter content.
+ * The view displaying Flutter content.
  *
- * This method may return |nil|, for instance in a headless environment.
+ * Some single-view APIs will eventually be replaced by their multi-view
+ * variant. During the deprecation period, the single-view APIs will coexist
+ * with and work with the multi-view APIs as if the other views don't exist.
+ * For backward compatibility, single-view APIs will always operate on this
+ * view, which is the first view assigned to the engine.
  *
- * The default view is a special view operated by single-view APIs.
+ * This method may return |nil|, for instance in a headless environment, or when
+ * multi-view is supported in the future and the compatible mode is disabled.
```

Comment:
Consider:

```suggestion
 * multi-view is supported in the future and single-view APIs are deprecated.
```

Reply:
I don't think this is correct. After phase 1, single-view APIs might or might not be deprecated immediately. And the compatible mode (i.e. implicit view enabled) can be enabled or disabled. They are not related. This field will be null when the compatible mode is disabled (or enabled but the implicit view is not assigned.)